app/design/frontend/Edu/Ultimate/Scandiweb_StoreFinder/templates/storefinder.phtml

```diff
@@ -0,0 +1,101 @@
+<?php
```

Comment:
This file seems very similar to /templates/storefinder.phtml, are they both needed?